app/open.js

```diff
@@ -111,18 +129,6 @@ define(['app/database', 'jquery', 'bootstrap', 'xlsx', 'parsley', 'bootstrap-sel
 		return result;
 	}
 
-	function process_wb(wb) {
-		var output = to_json(wb);
-		var sheetName = Object.keys(output)[0];
-
-		$.each(output[sheetName], function (i, item) {
-			var question = output[sheetName][i].question;
-			var answer = output[sheetName][i].answer;
-			db.executeQuery('addDatasetItem' , ['datasets1', question, answer, 'hint']);
-		})
-		db.close();
-	}
-
 	function handleFile(e) {
```

Comment:
The `handleFile(e)` part is part of the XLSX library code. We chose to only adapt the third party code in functionality to suit our needs; not in style.